Android/src/org/droidplanner/android/fragments/actionbar/ActionBarTelemFragment.java

```diff
@@ -311,11 +321,17 @@ else if (signalStrength >= 25)
     private void updateGpsTelem() {
         final Drone drone = getDrone();
 
+        final View popupView = gpsPopup.getContentView();
+        TextView satNoView = (TextView) popupView.findViewById(R.id.bar_gps_satno);
+        TextView hdopView = (TextView) popupView.findViewById(R.id.bar_gps_hdop);
```

Comment:
@funtastix if the `gps hdop` is displayed by default, then the `hdopView` should have its visibility set to `gone`, avoiding a repeat of the same information.